### PR TITLE
NAS-104714 / 12.0 / Remove zeroconf from SMB, add Service Announcement section to Network Configuration

### PR DESCRIPTION
**Pull request by erinix**
NAS-104714

### CODE REVIEW

**Comment by erinix** [comment]
could you please review the text and provide me with some tooltips @freenas/docs-team

File: src/app/helptext/network/configuration/configuration.ts [hunk 1]
```diff
@@ -64,5 +64,13 @@ hosts_tooltip : T('Additional hosts to be appended to <i>/etc/hosts</i>.\
 hostname_and_domain: T('Hostname and Domain'),
 gateway: T('Default Gateway'),
 nameservers: T('DNS Servers'),
-other: T('Other Settings')
+other: T('Other Settings'),
+service_announcement: T('Service Announcement'),
+
+netbios_placeholder: T('NetBIOS-NS'),
+netbios_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts]:
```suggestion
netbios_tooltip: T('Legacy NetBIOS name server. Advertises the SMB \
 Service <i>NetBIOS Name</i>. Could be required for legacy SMB1 clients \
 to discover the server (server appears in <i>Network Neighborhood</i>).'),
```

File: src/app/helptext/network/configuration/configuration.ts [hunk 2]
```diff
@@ -64,5 +64,13 @@ hosts_tooltip : T('Additional hosts to be appended to <i>/etc/hosts</i>.\
...
+mdns_placeholder: T('mDNS'),
+mdns_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts]:
```suggestion
mdns_tooltip: T('Multicast DNS. Uses the system <i>Hostname</i> to \
 advertise enabled and running services. For example, this controls if \
 the server appears under <i>Network</i> on MacOS clients.'),
```

File: src/app/helptext/network/configuration/configuration.ts [hunk 3]
```diff
@@ -64,5 +64,13 @@ hosts_tooltip : T('Additional hosts to be appended to <i>/etc/hosts</i>.\
...
+wsd_placeholder: T('WS-Discovery'),
+wsd_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts]:
```suggestion
wsd_tooltip: T('Uses the SMB Service <i>NetBIOS Name</i> to advertise \
 the server to WS-Discovery clients. This causes the computer appear in \
 the <i>Network Neighborhood</i> of modern Windows OSes.'),
```

**Pull request merged**